content/docs/rfcs/27/README.md

```diff
@@ -0,0 +1,98 @@
+---
+slug: 27
+title: 27/WAKU2-PEERS
+name: Waku v2 Client Peer Management Recommendations
+status: raw
```

Comment:
Probably draft assuming this is implemented?

Reply:
It is. So `draft` it is.